target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8065.dtsi

```diff
@@ -84,3 +31,88 @@
 		};
 	};
 };
+
+&opp_table0 {
+	compatible = "operating-points-v2-qcom-cpu";
+	nvmem-cells = <&speedbin_efuse>;
+
+	// On ipq8065 1.2 ghz freq is not present
```

Comment:
I would remove the comment and added it to the commit description, if you insist on the comment, then please use /**/ format.

Reply:
Think is better to put a comment in the dtss